code/src/reducers/todos.js

```diff
@@ -0,0 +1,37 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+const initialState = {
+  list: {
+    name: 'Test List',
+    items: [
+      {
+        description: "Call mummy",
+        done: false
+      }
+    ]
+  }
+}
+
+export const todos = createSlice({
+  name: "todos",
+  initialState: initialState,
+  reducers: {
+    addTodo: (state, action) => {
+      const { itemInfo } = action.payload
+      state.list.items.push(itemInfo)
+    },
```

Comment:
- Try adding a unique id to your todos that you can use instead of the index, eg. a timestamp. In Jennie's codealong from week 15 you can get a hint on how: https://github.com/Technigo/codealong-week15-20april/blob/master/src/reducers/fridge.js

- To get the timestamp to work, one idea is to add it to your items:
`state.list.items.push({ description: itemInfo.description, done: itemInfo.done, createdAt: moment() })`